src/common/meta/src/key/flow/flow_info.rs

```diff
@@ -215,6 +215,32 @@ impl FlowInfoManager {
             TxnOpGetResponseSet::decode_with(TxnOpGetResponseSet::filter(key)),
         ))
     }
+
+    /// Builds a update flow transaction.
+    /// It is expected that the `__flow/info/{flow_id}` IS ALREADY occupied.
+    /// Otherwise, the transaction will retrieve existing value and fail.
+    pub(crate) fn build_update_txn(
+        &self,
+        flow_id: FlowId,
+        flow_value: &FlowInfoValue,
+    ) -> Result<(
+        Txn,
+        impl FnOnce(&mut TxnOpGetResponseSet) -> FlowInfoDecodeResult,
+    )> {
+        let key = FlowInfoKey::new(flow_id).to_bytes();
+        let txn = Txn::new()
+            .when(vec![Compare::new(key.clone(), CompareOp::NotEqual, None)])
```

Comment:
ditto

Reply:
> ditto

I think here we can leave out cmp with previous value to allow `CREATE OR REPLACE` to replace previous flow even when flow's info value is exactly the same